docs/platforms/javascript/common/sourcemaps/uploading/angular-nx.mdx

```diff
@@ -53,7 +53,7 @@ If you are on an older version and you want to upload source maps we recommend u
 
 </Note>
 
-To upload source maps you have to configure an auth token.
+To upload source maps you have to configure an [Organization Auth Token](https://sentry.io/orgredirect/organizations/:orgslug/settings/auth-tokens/).
```

Comment:
what about this? (makes it sound a little bit less strict)
```suggestion
To upload source maps, configure an [Organization Auth Token](https://sentry.io/orgredirect/organizations/:orgslug/settings/auth-tokens/).
```

Reply:
hmm, not 100% sure, I'll defer this to @getsentry/docs folks for the best wording :D

Reply:
I am with @Lms24. You don't *have* to use an org auth token.

Reply:
Sorry I'm confused. It sounds like this is a requirement, but also like it's not? Can you clarify what the org auth token is used for? Can users still upload source maps manually without the org auth token?

Reply:
So you can either use an org auth token (which is the recommend way and what we generally propose now everywhere), which is easy to set up in that there are no config options.
Or you can also use a personal auth token, which has to be configured correctly.

What if we go with something like this then:

> To upload source maps you have to configure an Organization Auth Token.
> 
> Alternatively, you can also use a Personal Auth Token with the X, Y and Z scopes (TBD: what are these exactly again now?)

Reply:
I like that suggestion 👍 

If we want to recommend using org auth tokens as opposed to personal auth tokens, we can also say

> To upload source maps, you should first configure an Organization Auth Token.

Reply:
> what are these exactly again now?

- Project read & write
- Release admin

(I think)

Reply:
What about `Organization`? 🤔 
> Manage Organizations, resolve IDs, retrieve Repositories and Commits

but I'll adjust it, we can still change it if something is missing!